ui/app/[locale]/(authorized)/kafka/[kafkaId]/@activeBreadcrumb/consumer-groups/[groupId]/page.tsx

```diff
@@ -1,22 +1,44 @@
 import { KafkaConsumerGroupMembersParams } from "@/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/KafkaConsumerGroupMembers.params";
 import { BreadcrumbLink } from "@/components/Navigation/BreadcrumbLink";
-import { BreadcrumbItem } from "@/libs/patternfly/react-core";
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Tooltip,
+} from "@/libs/patternfly/react-core";
+import { HomeIcon } from "@/libs/patternfly/react-icons";
+import { useTranslations } from "next-intl";
 
 export default function ConsumerGroupsActiveBreadcrumb({
   params: { groupId, kafkaId },
 }: {
   params: KafkaConsumerGroupMembersParams;
 }) {
-  return [
-    <BreadcrumbLink
-      key={"cg"}
-      href={`/kafka/${kafkaId}/consumer-groups`}
-      showDivider={true}
-    >
-      Consumer groups
-    </BreadcrumbLink>,
-    <BreadcrumbItem key={"cgm"} showDivider={true} isActive={true}>
-      {decodeURIComponent(groupId) === "+" ? <i>Empty Name</i> : groupId}
-    </BreadcrumbItem>,
-  ];
+  const t = useTranslations("breadcrumbs");
+  return (
+    <Breadcrumb>
+      <BreadcrumbItem key="home" to="/" showDivider>
+        <Tooltip content={t("view_all_kafka_clusters")}>
+          <HomeIcon />
+        </Tooltip>
+      </BreadcrumbItem>
+      <BreadcrumbItem
+        key="overview"
+        to={`/kafka/${kafkaId}/overview`}
+        showDivider
+      >
+        {t("overview")}
+      </BreadcrumbItem>
+      <BreadcrumbLink
+        key={"cg"}
+        href={`/kafka/${kafkaId}/consumer-groups`}
+        showDivider={true}
+      >
+        {t("consumer_groups")}
+      </BreadcrumbLink>
+      ,
```

Comment:
```suggestion
```

Reply:
Apologies for the typo. removed the unwanted comma.